kalite/distributed/management/commands/kaserve.py

```diff
@@ -53,7 +53,7 @@ class Command(BaseCommand):
             '--pidfile',
             action='store',
             dest='pidfile',
-            default=os.path.join(settings.PROJECT_PATH, "runcherrypyserver.pid"),
+            default=os.path.join(settings.USER_DATA_ROOT, "runcherrypyserver.pid"),
```

Comment:
This may require changes in the windows installer; but I guess we'll be refactoring the windows installer to use the setup.py stuff anyway?